easybuild/easyconfigs/s/Singularity/Singularity-3.5.3-GCC-8.3.0.eb

```diff
@@ -0,0 +1,63 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage: 		https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::		Robert Qiao <rob.qiao@flinders.edu.au>
+# Software License::	Apache v2.0 and Custom
+#
+# Notes::
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'Singularity'
+version = '3.5.3'
+
+github_account = 'sylabs'
+
+homepage = 'https://sylabs.io'
+description = """Singularity is a portable application stack packaging and runtime utility."""
+
+toolchain = {'name': 'GCC', 'version': '8.3.0'}
+
+sources = [{
+           'filename': 'v%(version)s.tar.gz',
+           'git_config': {
+               'url': 'https://github.com/%(github_account)s',
+               'repo_name': '%(namelower)s',
+               'tag': 'v%(version)s',
+               'recursive': True, },
+           }]
+checksums = ['f057a15d28dae27b2e04cd1dd6a26e18daab41f89f16f07594302c2bcbb263f4']
+
+builddependencies = [
+    ('Autotools', '20180311'),
+    ('libarchive', '3.4.2'),
+    ('Go', '1.14', '', True),
+]
+
+configure_cmd = "cd '%(builddir)s/%(namelower)s-%(version)s' && "
+configure_cmd += "echo '%(version)s' > VERSION && "
+configure_cmd += "./mconfig -p '%(installdir)s' -b ./buildtree"
+
+build_cmd = "cd ./buildtree && "
+build_cmd += "make"
+
+install_cmd = "cd '%(builddir)s/%(namelower)s-%(version)s/buildtree' && "
+install_cmd += "make install"
```

Comment:
@robqiao it's not a good idea to overwrite the _cmd variables unless really necessary. I suggest to change this as follows:
```
start_dir = '%(namelower)s-%(version)s'

configure_cmd = "echo '%(version)s' > VERSION && "
configure_cmd += "./mconfig -p '%(installdir)s' -b ./buildtree"

prebuildopts = "cd buildtree && "

preinstallopts = "cd buildtree && "
```